packages/rest-explorer/templates/index.html.ejs

```diff
@@ -5,7 +5,9 @@
 <head>
   <meta charset="UTF-8">
   <title>LoopBack API Explorer</title>
-  <link rel="stylesheet" type="text/css" href="./swagger-ui.css">
+  <!-- customize the swagger-ui theme, the corresponding asset path is ./themes/-->
+  <!-- FIXME: the theme path should be provided as a template variable -->
+  <link rel="stylesheet" type="text/css" href="<%- swaggerThemePath %>">
```

Comment:
Should we perhaps add a new server endpoint that will serve the CSS file, so that the HTML file can keep using the same static path?

```suggestion
  <link rel="stylesheet" type="text/css" href="./swagger-theme.css">
```

The controller method implementing `swagger-theme.css` can use `swagerThemePath` to locate the file to read & return back in the response.

I guess my main concern is that with your current proposal, how are you going to ensure that `swaggerThemePath` is pointing to a file that is served by the static middleware?

Reply:
(I think my comment above can be discarded, but I am keeping here to show other possible approaches how to solve this feature.)

Reply:
> how are you going to ensure that swaggerThemePath is pointing to a file that is served by the static middleware

It's a valid question :)
The user needs to make sure the `swaggerThemeFile` points to the right css file. 
E.g. todo app has `/public` folder set up as the default home page with url `/`. To load the css file in `/public/theme-newspaper.css`, use

```ts
{swaggerThemeFile: '/theme-newspaper.css'}
```